models/DataObject/ClassDefinition/Data/Link.php

```diff
@@ -64,11 +64,7 @@ class Link extends Model\DataObject\ClassDefinition\Data
      */
     public function getDataForResource($data, $object = null, $params = [])
     {
-        if ($data instanceof DataObject\Data\Link and isset($data->object)) {
-            unset($data->object);
-        }
-
-        if ($data) {
+        if ($data instanceof DataObject\Data\Link) {
```

Comment:
@blankse I guess that's a leftover of #3706 - that shouldn't change, because it would break things here.

Reply:
@brusch The `isset($data->object)` makes no sense. There is neither a public object attribute nor a __isset function.

Also should check if is a Link object, because the method calls afterwards will fail.

What did you think will break?

Reply:
There is a setObject and getObject function but no property for that.

Reply:
Oh, I see. I will make another PR for this :sweat_smile:

Reply:
The setObject and getObject don't use the object attribute anymore (https://github.com/pimcore/pimcore/commit/a67d648e7ef0374e74a9c2b1e2046d92d7e437d8#diff-a67f6fdbaa4e6437319bf7fa07ac1ae4)